sphinx-docs/screenshot.py

```diff
@@ -203,6 +203,7 @@ class Screenshot(Image):
     option_spec['user-role'] = _parse_user_role
     option_spec['navigation-steps'] = _parse_nav_steps
     option_spec['focus'] = _parse_focus
+    option_spec['registered'] = lambda x: directives.choice(x, ("true", "false"))
```

Comment:
However, failure to specify will default to `False`, which was the behavior before these changes -- screenshots were taken as though the device was not registered. In other words, these changes don't backwardly break anything.